targets/TARGET_NUVOTON/TARGET_NUC472/crypto/crypto-misc.c

```diff
@@ -0,0 +1,159 @@
+/* mbed Microcontroller Library
```

Comment:
Is it possible to share this file (or at least the common portions) between NUC472 and M480? This would make maintenance easier.

Reply:
@Patater As there is another target which supports crypto, we'll consider refactoring crypto HAL.